devices.js

```diff
@@ -5961,6 +5961,37 @@ const devices = [
             await configureReporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['SZ-DWS04N_SF'],
+        model: 'SZ-DWS04N_SF',
```

Comment:
Do you have a URL to this product, is it the same as: https://www.zigbee2mqtt.io/devices/SZ-DWS04.html ?

Reply:
Yes, I have both, SZ-DWS04N_SF and SZ-DWS04 are esthetically and functionally identical, probably two different revisions of the same device

Reply:
As the definition is also identical, and as googling for `SZ-DWS04N_SF` doesn't yield any results (doesn't seem to be sold under this model name, so probably still under `SZ-DWS04N`) I will merge the two definitions.